<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptingWindow</class>
 <widget class="QDialog" name="ScriptingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripting Interface</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Loaded Scripts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listLoadedScripts"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadScript">
       <property name="text">
        <string>Load New Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNewScript">
       <property name="text">
        <string>Create New Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveScript">
       <property name="text">
        <string>Remove Current Script</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0,0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Script Editing Window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtScriptSource">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnSaveScript">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRevertScript">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRecompile">
         <property name="text">
          <string>Recompile</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Current Scripting Errors</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listErrors"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
